### PR TITLE
20240210 3 addition info

### PR DESCRIPTION
**Pull request by m1ra9e**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ 
+## [Unreleased]
+
+### Added
+
+### Changed
+
+### Fixed
+
+## [1.0.0] - XXXX-XX-XX
```
Comment (m1ra9e) [inline CHANGELOG.md:16]: Am I correct? : 1 major version

Comment (k-l-e-v-e-r) [reply CHANGELOG.md:16]: yes

Comment (k-l-e-v-e-r) [reply CHANGELOG.md:16]: without date?

Comment (m1ra9e) [reply CHANGELOG.md:16]: yes

**Pull request merged**